File: dataset/output-bk/MISTRAL_codestral-latest/Task44_MISTRAL_codestral-latest.cpp
#include <iostream>

int main() {
    double num;

    std::cout << "Enter a number: ";

    if (std::cin >> num) {
        std::cout << "The square of the number is: " << num * num << std::endl;
    } else {
        std::cout << "Invalid input. Please enter a number." << std::endl;
    }

    return 0;
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task44_MISTRAL_codestral-latest.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number: ");

        if (scanner.hasNextDouble()) {
            double num = scanner.nextDouble();
            System.out.println("The square of the number is: " + (num * num));
        } else {
            System.out.println("Invalid input. Please enter a number.");
        }

        scanner.close();
    }
}